STARTERKIT/gulpfile.js

```diff
@@ -240,12 +232,37 @@ gulp.task('clean:styleguide', function () {
 
 // Clean CSS files.
 gulp.task('clean:css', function () {
-  return del([
+  return $.del([
     options.theme.css + '**/*.css',
     options.theme.css + '**/*.map'
   ], {force: true});
 });
 
+// Converts module names to absolute paths for easy imports.
+function sassModuleImporter(url, file, done) {
```

Comment:
Please do not move code around the file, edit it where it lives

Reply:
structure will change more. actually i think this function will be changed with some module.